src/components/Tool/FixPhoneNumbers/Contact.tsx

```diff
@@ -206,7 +202,9 @@ const Contact: React.FC<Props> = ({
                               </Typography>
                             </Hidden>
                             <Typography display="inline">
-                              {`${phoneNumber.source} (${phoneNumber.date})`}
+                              {`${'Source'} (${DateTime.fromISO(
+                                phoneNumber.updatedAt,
+                              ).toLocaleString(DateTime.DATE_SHORT)})`}
```

Comment:
The source field is missing in GQL, so this will be fixed when it is added

Reply:
Ah did you disable deleting because the source isn't currently provided? That makes sense if that is the case.

Reply:
If this is a TODO item, I think it would be good to add a comment saying when it can be fixed (and reference any relevant PRs/tickets if applicable).